utils/email-content-parser.test.ts

```diff
@@ -0,0 +1,138 @@
+import { describe, it, expect } from "vitest";
+import { cleanEmailContent } from "./email-content-parser";
```

Comment:
remove both the method and the test to the `server` top level directory. We will eventually delete the utils directory